### PR TITLE
Features/port courses

### PR DESCRIPTION
**Pull request by hikes101**
All content of the course page is ported. The layout is very basic so far but I will take care of that later in another branch.

### CODE REVIEW

File: data/courses.json [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+      "furtherLink": "",
+      "time": ""
+    },
+    {
+      "name": "Lerngruppen zu euren Themen",
+      "img": "img/courses/Lerngruppen.png",
+      "header": "",
+      "description": "Für Menschen aller Altersgruppen ist die Lernwerkstatt ein ideeller und realer Raum zum Bearbeiten gemeinsamer Themen. Das kann in Form von Workshops, einmaligen Abendveranstaltungen, fortlaufenden Gruppen, Diskussionen und vielem mehr sein. Ebenso breit gefächert ist das Themenspektrum: Mathematik & Naturwissenschaften, Programmieren & Multimedia, Sprachen, Philosophie, alternative Bildung, u.v.m.",
+      "additional": "",
+      "furtherLink": "",
```
Comment (jarjan) [inline data/courses.json]: If no link better add just hashtag.
```suggestion
       "furtherLink": "#",
```

File: src/routes/courses.js [hunk 2]
```diff
@@ -1,8 +1,16 @@
 const express = require("express");
+const path = require("path");
+const fs = require("fs");
 const router = express.Router();
 
+const rootPath = path.join(__dirname, "../", "../");
+const coursesPath = path.join(rootPath, "data/courses.json");
```
Comment (jarjan) [inline src/routes/courses.js]:
```suggestion
 const coursesPath = path.join(__dirname, "../data/courses.json");
```
and you can remove `rootPath`

File: views/courses.handlebars [hunk 3]
```diff
@@ -1,3 +1,32 @@
...
+          <h1>Kurse &amp; Lernbegleitung</h1>
+          <p>Unser Lernangebot</p>
+        </div>
+        <div class="col-12 col-sm-5 align-self-center">
+          <img src="https://res.cloudinary.com/hrscywv4p/image/upload/c_fill,fl_lossy,h_200,w_200,f_auto,g_faces:auto,q_auto/v1/253787/6b3c1ba6-333a-4dfb-80b1-686bd4b9e0db_s4dlam.png" class="img-fluid rounded-circle" alt="Lernwerkstatt Logo">
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="container-fluid">
```
Comment (jarjan) [inline views/courses.handlebars]: Maybe don't use fluid container?
```suggestion
   <div class="container">
```

File: data/courses.json (hunk 1 above)
Comment (hikes101) [reply data/courses.json]: Done

File: src/routes/courses.js (hunk 2 above)
Comment (hikes101) [reply src/routes/courses.js]: Done

File: static/css/style.css [hunk 4]
```diff
@@ -64,6 +64,21 @@ body {
   justify-content: center;
 }
 
+/* COURSES PAGE */
+.courses-box {
```
[the remark this answers is not in the document]
Comment (hikes101) [reply static/css/style.css]: Ok, I changed it

**Pull request merged**